src/app/components/order-page-components/order-page/order-page.component.ts

```diff
@@ -0,0 +1,102 @@
+import { Component, HostListener, OnInit } from '@angular/core';
+import { AbstractControl, FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { CountriesCities } from '../../../interfaces/countries-cities';
+import { ProductCard } from '../../../interfaces/product-card';
+import { BreadcrumbsService } from '../../../services/breadcrumbs.service';
+import { countriesAndCities } from './countries-cities-constant';
+
+@Component({
+  selector: 'app-order-page',
+  templateUrl: './order-page.component.html',
+  styleUrls: ['./order-page.component.scss']
+})
+
+@HostListener('window:resize', ['$event'])
```

Comment:
Did you use this event somewhere?

Reply:
HostListener decorator has been removed due to no usage.